src/Behat/Symfony2Extension/Context/Argument/ServiceArgumentResolver.php

```diff
@@ -60,10 +60,6 @@ public function resolveArguments(ReflectionClass $classReflection, array $argume
      */
     private function resolveArgument($argument)
     {
-        if (is_array($argument)) {
-            return array_map(array($this, 'resolveArgument'), $argument);
```

Comment:
Very strange, I wasn't aware that was in the diff before I committed. I've re-instated that chunk, re-tested my scenarios locally and it's running. Unit tests aren't running on my machine right now so I'm relying on Travis.